File: MyPortal.Database/Search/StudentSearch.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace MyPortal.Database.Search
{
    public class StudentSearch : PersonSearch
    {
        public Guid? RegGroupId { get; set; }
        public Guid? YearGroupId { get; set; }
        public Guid? HouseId { get; set; }
        public Guid? SenStatusId { get; set; }
    }
}

File: MyPortal-App/src/app/_models/People/Students/student-search.ts
import { PersonSearch } from './../person-search';
export class StudentSearch extends PersonSearch {
    status: string;
    curriculumGroupId: string;
    regGroupId: string;
    yearGroupId: string;
    houseId: string;
    senStatusId: string;

    constructor(){
        super();
        this.status = '1';
        this.curriculumGroupId = null;
        this.regGroupId = null;
        this.yearGroupId = null;
        this.houseId = null;
        this.senStatusId = null;
    }
}
